### ISSUE TITLE
User-guide: add doc to describe how to tune ironic before deploying

### ISSUE THREAD

**Issue by fmuyassarov** [metal3-io/metal3-docs#233]
Describe:
- how to tune Ironic parameters before deploying. Example, how to enable a certain features like fast track before ironic is deployed
- what is the format we pass or manipulate those parameters. As an example, in dev-env we use configMap

/kind documentation
/help

**Comment by tuminoid** [comment]
This will probably be describing IRSO mechanics as its going to be handling Ironic installation in future.

**Comment by dtantsur** [comment]
Yep, it will be a part of the IrSO user guide (which I hope to write before EOY)

**Comment by Rozzii** [comment]
> Yep, it will be a part of the IrSO user guide (which I hope to write before EOY)

I will add this to the MVP milestone, just to not forget about it.
